composer.json

```diff
@@ -80,9 +78,6 @@
         }
     },
     "extra": {
-        "branch-alias": {
-            "dev-master": "3.0.x-dev"
-        },
```

Comment:
this means somebody installing `^3.0@dev` not longer composer can resolve itself. I my personal recommendation would be rename `master` to `3.0` or `3.x` that avoids branch aliases and composer directly know what version a branch suppose to be without you need to handle or update branch aliases.

Reply:
It was about the first time package splitting. Yii2 -> Yii3
Now just use `dev-master` if you want to get master upstream.
But I suggest to use tags. Now the package is unstable and there's one way to install it is to use `dev-master` branch.

Reply:
Aware of that, but still a `^3.0@dev` would atleast make sure semver is followed. Renaming master -> 3.x or 3.0 would tell composer automatically what future version is and allow projects do `^3.0@dev`. Also if you tag in future the 3.0.0 will a branch named `3.0` or `3.x` will composer help to install that branch when somebody allow stability dev. With a branch named master without a branch alias composer can not detect the version. Its also one of the cases why example Symfony and we @sulu use version name as branches for all our packages.